tests/components/yeelight/test_config_flow.py

```diff
@@ -230,32 +234,55 @@ async def test_options(hass: HomeAssistant):
         await hass.async_block_till_done()
 
     config = {
+        CONF_NAME: NAME,
         CONF_MODEL: "",
         CONF_TRANSITION: DEFAULT_TRANSITION,
         CONF_MODE_MUSIC: DEFAULT_MODE_MUSIC,
         CONF_SAVE_ON_CHANGE: DEFAULT_SAVE_ON_CHANGE,
         CONF_NIGHTLIGHT_SWITCH: DEFAULT_NIGHTLIGHT_SWITCH,
     }
-    assert config_entry.options == {
-        CONF_NAME: "",
-        **config,
-    }
+    assert config_entry.options == config
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)